config.yaml

```diff
@@ -0,0 +1,420 @@
+        region: &REGION us-central1
+   
+    - type: subnetwork_v1
+      name: &Production_Data 
+      properties:
+        network: $(ref.first-deployment-network.selfLink)
+        ipCidrRange: &NETWORK_PROD_DATA 192.168.2.0/24
+        region: *REGION 
+
+    - type: subnetwork_v1
+      name: &Production_Shared
+      properties:
+        network: $(ref.first-deployment-network.selfLink)
+        ipCidrRange: &NETWORK_PROD_SHARED 192.168.3.0/24
+        region: *REGION 
+
+    - type: subnetwork_v1
+      name: &Staging
+      properties:
+        network: $(ref.first-deployment-network.selfLink)
+        ipCidrRange: &NETWORK_STAGING 192.168.4.0/24
+        region: *REGION    
+
+    - type: subnetwork_v1
+      name: &Development
+      properties:
+        network: $(ref.first-deployment-network.selfLink)
+        ipCidrRange: &NETWORK_DEV 192.168.5.0/24
+        region: *REGION     
+
+#Defining IGM 
+  - type: gce_instancetemplate_v1.py
+    name: Magento 
+    properties:
+      machineType: n1-standard-4
+      image_url: "projects/centos-cloud/global/images/family/centos-7"
+      networkInterfaces:
+        - accessConfigs:
+            - {name: external-nat, type: ONE_TO_ONE_NAT}
+      disks:
+        - deviceName: 'boot'
+        - boot: true
+        - autoDelete: false
+
+    - type: igm_v1
+      name: Mgento
+      properties:
+        region: *REGION
+        targetSize: 1
+        instanceTemplate:
+
+#Autoscale group for MIG  
+    - type: autoscaler_v1.py
+      name: IGM-Magento
+      properties:
+        region: *Region
+        autoscalingPolicy.minNumReplicas: 1
+        autoscalingPolicy.maxNumReplicas: 4
+        target: $(ref.Magento.safeLink)
+
+#Load balancer
+    - type: httploadbalancer_v1.py
+      name: httploadbalancer
+      backendProperties:
+        backends:
+          - group: $(ref.IGM-Magento.safeLink)
+      healthCheckProperties:
+        tls: false 
+        port: 80
+
+#Instances:
+    - type: gce_instance_v2.py
+      name: Varnish
+      properties:
+      disks:
+        - deviceName: 'boot'
+          boot: true
+          autoDelete: false
+          initializeParams:
+            sourceImage: projects/centos-cloud/us-central1-a/images/centos-7
+            diskSizeGb: 60
+        machineType: n1-standard-2
+        networkInterfaces:
+          - network: *NETWORK 
+            subnetworks: *Production Web
+            
+
+    - type: gce_instance_v2.py
+      name: NAT Gateway
+      properties: 
+        networkInterfaces:
+          - network: *NETWORK 
+            subnetworks: *Production_Web
+        disks:
+            - deviceName: 'boot'
+              boot: true
+              sizeGb: 60
+              sourceImage: projects/centos-cloud/global/images/family/centos-7
+              type: project/zones/us-central1-a/diskTypes/pd-Standard             
+          machineType: zones/us-central1-a/machineTypes/n1-standard-4
+          disks[].autoDelete: False
```

Comment:
Invalid property, `autoDelete` is a property of a disk resource.

Reply:
Need to discuss